src/etc/inc/filter.inc

```diff
@@ -1702,6 +1702,19 @@ function filter_nat_rules_automatic_tonathosts($with_descr = false) {
 		$tonathosts[] = "{$config['ipsec']['client']['pool_address']}/{$config['ipsec']['client']['pool_netbits']}";
 		$descriptions[] = gettext("IPsec client");
 	}
+	if ((isset($config['ipsec']['client']['enable'])) &&
+		(isset($config['ipsec']['mobilekey']))) {
+		foreach ($config['ipsec']['mobilekey'] as $key) {
+			if (in_array("{$key['pool_address']}/{$key['pool_netbits']}", $tonathosts)) {
+				continue;
+			}
+			if ((!empty($key['pool_address'])) &&
+				(!empty($key['pool_netbits']))) {
```

Comment:
Second level indent should be 4 spaces instead of a tab. While here, please remove extra parenthesis